### ISSUE TITLE
Add a tabbed navigation to move between `vizro-core` and `vizro-ai` docs

### ISSUE THREAD

**Issue by stichbury** [mckinsey/vizro#390]
The Kedro docs (https://docs.kedro.org) has a tabbed navigation.

<img width="1089" alt="image" src="https://github.com/mckinsey/vizro/assets/5180475/5527861a-aba0-45d8-8d36-977226c098f8">

Why can't we have something similar? This shouldn't be a major piece of work and will simply enable users to move between https://vizro.readthedocs.io/en/stable/ and https://vizro.readthedocs.io/projects/vizro-ai/en/latest/ at a top level.

We did this in Kedro in the `layout.html` https://github.com/kedro-org/kedro/blob/main/docs/source/_templates/layout.html

Anyone better at front end design and development will be able to add to this ticket to explain how to make it work.

Ideally we should have some design input but I know we are already waiting on that for other tickets so perhaps we could mock this up in code and then add it to the queries we have for Design.

Suitable for help from anyone with front-end skills, no prior knowledge of Vizro is required, so I'm marking as "Help Wanted" and "Good First Issue"

**Comment by stichbury** [comment]
Note to add: We could also add another tab to point into the contribution documentation since this will be the same for all packages in the monorepo and will have one single source of docs (plus it's different kind of docs -- contributor content not user content -- so makes sense to separate it in the nav).

**Comment by maxschulz-COL** [comment]
@stichbury Is this not already done? Or do you mean a tab outside the other content tabs?

**Comment by stichbury** [comment]
Yes, it's now done (this is from March) 😅
